index.html

```diff
@@ -1,44 +1,225 @@
+                <div class="projects">
+                    <h2 class="project-tittle">Facebook 360</h2>
+                    <ul class="project-info">
+                        <li class="client">FACEBOOK</li>
+                        <li><img class="imgPoint" src="images/header/Counter.png" alt="point"></li>
+                        <li>Full Stack Dev</li>
+                        <li><img class="imgPoint" src="images/header/Counter.png" alt="point"></li>
+                        <li>2015</li>
+                    </ul>
+                    <p class="primary-text">Exploring the future of media in Facebook's first Virtual Reality app; a
+                        place
+                        to discover and enjoy 360 photos and videos on Gear VR.</p>
+                    <ul class="categories">
+                        <li>html</li>
+                        <li class="ruby">Ruby on rails</li>
+                        <li>css</li>
+                        <li>javaScript</li>
+                    </ul><button class="button" id="3">See Project</button>
+                </div>
+            </div>
+            <div class="works-2">
+                <div class="project-image"><img class="image" src="images/header/desktop/d.png" alt="my first project"></div>
+                <div class="projects">
+                    <h2 class="project-tittle">Uber Navigation</h2>
+                    <ul class="project-info">
+                        <li class="client">Uber</li>
+                        <li><img class="imgPoint" src="images/header/Counter.png" alt="point"></li>
+                        <li>Lead Developer</li>
+                        <li><img class="imgPoint" src="images/header/Counter.png" alt="point"></li>
+                        <li>2018</li>
+                    </ul>
+                    <p class="primary-text">A smart assistant to make driving more safe, efficient, and fun by unlocking
+                        your most expensive computer: your car.</p>
+                    <ul class="categories">
+                        <li>html</li>
+                        <li class="ruby">Ruby on rails</li>
+                        <li>css</li>
+                        <li>javaScript</li>
+                    </ul><button class="button" id="4">See Project</button>
+                </div>
+            </div>
+        </div>
+        <section class="About_me" id="about_me">
+            <div class="about_me_full">
+                <h2 class="about_myself_h2">About Myself</h2>
+                <p class="about_myself_p"> Hello I’m a software developer! I can help you build a product , feature or
+                    website Look
+                    through some of my work and experience! If you like what you see and have a project you need coded,
+                    don’thestiate to contact me. </p>
+                <p class="about_connect_2">LET'S CONNECT</p>
```

Comment:
- [ ] Kindly try to follow the design on Figma as much as possible. For the `about me` section, the `background-color` should be white. The section should have `border-radius` on the top right corner. The languages (HTML, CSS, JavaScript) should be aligned in a column (the picture is above the text) and not in a row. Also, kindly fix the path to the arrow images, they're not showing (hint: you can use `font awesome` for icons instead of images)

## Your design:
![image](https://user-images.githubusercontent.com/50721479/194709307-52e74cec-457b-4d6a-9da2-c79345fec0a4.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/194709315-a16265d1-9377-4f13-b8a3-1eda3dbfeade.png)